Content.Shared/Chemistry/Components/Solution.cs

```diff
@@ -167,34 +167,41 @@ public FixedPoint2 GetReagentQuantity(string reagentId)
             return FixedPoint2.New(0);
         }
 
-        public void RemoveReagent(string reagentId, FixedPoint2 quantity)
+        /// <summary>
+        ///     Attempts to remove an amount of reagent from the solution.
+        /// </summary>
+        /// <param name="reagentId">The reagent to be removed.</param>
+        /// <param name="quantity">The amount of reagent to remove.</param>
+        /// <returns>How much reagent was actually removed. Zero if the reagent is not present on the solution.</returns>
+        public FixedPoint2 RemoveReagent(string reagentId, FixedPoint2 quantity)
         {
             if(quantity <= 0)
-                return;
+                return 0;
```

Comment:
not sure bout this, but wouldn't it be ever so slightly faster to use the static FixedPoint2.Zero, rather than creating a new fixed point instance?

Reply:
Oh ye, I completely forgot `FixedPoint2.Zero` was a thing. Done now